Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -104,10 +104,10 @@ extension Option where Value: ExpressibleByArgument {
 public enum SingleValueParsingStrategy {
   /// Parse the input after the option. Expect it to be a value.
   ///
-  /// For input such as `--foo foo` this would parse `foo` as the
-  /// value. However, for the input `--foo --bar foo bar` would
-  /// result in a error. Even though two values are provided, they don’t
-  /// succeed each option. Parsing would result in an error such as
+  /// For inputs such as `--foo foo`, this would parse `foo` as the
+  /// value. However, the input `--foo --bar foo bar` would
+  /// result in an error. Even though two values are provided, they don’t
+  /// succeed each option. Parsing would result in an error such as the following.
```

Comment:
Let's use a colon when introducing a code sample this way.
```suggestion
  /// succeed each option. Parsing would result in an error such as the following:
```